### PR TITLE
updated Go/iris

### PR DESCRIPTION
**Pull request by chunariov**
Fully updated Go/iris, which passes all tests.

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@chunariov Thanks for your contribution.

We don't want to add 800 vendor files in this repo. Those should all be pulled from github as one of the docker commands using `go get` like https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Go/martini/martini.dockerfile#L10

**Comment by chunariov** [comment]
Vendor files are deleted

File: frameworks/Go/iris/iris.dockerfile [hunk 1]
```diff
@@ -1,14 +1,11 @@
-FROM golang:1.10.1
+FROM golang:latest
```
Comment (nbrady-techempower) [inline frameworks/Go/iris/iris.dockerfile]: Please keep this at a specific version number

Comment (chunariov) [reply frameworks/Go/iris/iris.dockerfile]: Done

**Pull request merged**